libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -1293,4 +798,34 @@ - (void)userChanged:(SFUserAccountChange)change {
                                                       userInfo:@{ SFUserAccountManagerUserChangeKey : @(change) }];
 }
 
++ (void)setAccountPersisterClass:(nullable Class) persisterClass {
+    AccountPersisterClass  = persisterClass;
+}
+
+- (id <SFUserAccountPersister>)accountPersister {
+    return accountPersister;
+}
+
+- (void)reload {
+    [accountsLock lock];
+
+    if (AccountPersisterClass)
```

Comment:
See my notes on the `reload` call in `init`.  It'd be nice if the consumer had more flexibility in configuring the account persister used in the account management.